examples/demo/src/reviews/StarRatingInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Rating, Typography, styled } from '@mui/material';
+import Icon from '@mui/icons-material/Stars';
+import { InputProps, useInput, useTranslate } from 'react-admin';
+
+const StyledRating = styled(Rating)({
+    '& .MuiRating-iconFilled': {
+        color: '#000',
+    },
+    '& .MuiRating-iconHover': {
+        color: '#000',
+    },
+});
+
+const StarRatingInput = (props: InputProps) => {
+    const { name = 'resources.reviews.fields.rating' } = props;
+    const { field } = useInput(props);
+    const translate = useTranslate();
+
+    return (
+        <Box
+            sx={{
+                flexDirection: 'column',
+                marginBottom: '1rem',
+            }}
```

Comment:
`Box` supports inline style props, I find it more legible. Besides, you're already using that capability for `display`.

```jsx
<Box  flexDirection="column" marginBottom="1rem" display="flex">
```